tests/phpunit/tests/term/getTermBy.php

```diff
@@ -209,17 +211,17 @@ public function test_query_should_not_contain_order_by_clause() {
 	 * @ticket 21760
 	 */
 	public function test_query_should_contain_limit_clause() {
-		global $wpdb;
-
 		$term_id = $this->factory->term->create(
 			array(
 				'name'     => 'burrito',
 				'taxonomy' => 'post_tag',
 			)
 		);
-		$found   = get_term_by( 'name', 'burrito', 'post_tag' );
+		add_filter( 'terms_pre_query', array( $this, 'get_query_from_filter' ), 10, 2 );
+		$found = get_term_by( 'name', 'burrito', 'post_tag' );
+		remove_filter( 'terms_pre_query', array( $this, 'get_query_from_filter' ) );
```

Comment:
```suggestion
```

Automatically torn down. Nit pick.

Reply:
There are lots of example where filters are removed in tests. 
https://github.com/WordPress/wordpress-develop/blob/2648a5f984b8abf06872151898e3a61d3458a628/tests/phpunit/tests/comment.php#L1096-L1098

I don't see how this hurts.